docs/4.1/layout/overview.md

```diff
@@ -70,13 +70,12 @@ Bootstrap primarily uses the following media query ranges—or breakpoints—in
 Since we write our source CSS in Sass, all our media queries are available via Sass mixins:
 
 {% highlight scss %}
-@include media-breakpoint-up(xs) { ... }
 @include media-breakpoint-up(sm) { ... }
 @include media-breakpoint-up(md) { ... }
 @include media-breakpoint-up(lg) { ... }
 @include media-breakpoint-up(xl) { ... }
 
```

Comment:
Let's insert an example then to back up that `xs` comment I suggested earlier.

Reply:
Added example, hope I interpreted the comment correctly.